File: HW01/CPP/src/Soccerball.cpp
#include "../include/Soccerball.h"
#include <iostream>
#include <string>
using namespace std;

Soccerball::Soccerball(string material, double price) : OutdoorItem(price){
    this->material = material;
    this->price = price;
}

void Soccerball::display() {
    cout << "$" << this->price << " - This soccerball is made of " << this->material;
}

File: HW01/PYTHON/project/Soccerball.py
from .OutdoorItem import OutdoorItem

class Soccerball(OutdoorItem):
    def __init__(self, price, material):
        super().__init__(price)
        self.material = material
             
    def display(self):
        print(f"${self.price} - This soccerball is made of {self.material}", end="")